app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,37 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: [:create, :destroy]
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build micropost_params
+
+    if @micropost.save
+      flash[:success] = t ".created"
+      redirect_to root_url
+    else
+      @feed_items = current_user.feed.page params[:page]
+      render "static_pages/home"
```

Comment:
tại sao dùng render

Reply:
Sử dụng render vì khi micropost không được lưu thì không cần gửi lại request để load lại toàn bộ các bản ghi nữa, ở đây render chỉ cần tạo response load view static_pages/home